praw/models/reddit/mixins/replyable.py

```diff
@@ -4,7 +4,7 @@
 
 from typing import TYPE_CHECKING
 
-from ....const import API_PATH
+from praw.const import API_PATH
 
 if TYPE_CHECKING:  # pragma: no cover
```

Comment:
We should add this pattern to the coverage ignore.

Reply:
Is that trivial to do?

Reply:
I think I added it to Async PRAW. Either in pyproject.toml or coveragerc

Reply:
Yes, it's working properly. I'll clean up the prama lines.

Reply:
Fixed.